README.md

```diff
@@ -186,6 +186,21 @@ Does not depend on any other roles
 
 > **Note:** As the role will need to know the remote used platform (32 or 64 bits), you must set `gather_facts` to `true` in your play.
 
+## Requirements for MikroTik clients
+MikroTik routers (https://mt.lv) doesn't support some OpenVPN features: UDP mode, LZO compression, TLS authentication (according to https://wiki.mikrotik.com/wiki/Manual:Interface/OVPN).
+
+So here is example which additional variables you may have in playbook if your clients will be Mikrotik routers (also if you already have standart udp configuration, create a separate config file with variables below and this config wouldn't affect working users):
```

Comment:
Typo in `standart` (I think you mean standar*d*)

Reply:
```suggestion
So here is example which additional variables you may have in playbook if your clients will be Mikrotik routers (also if you already have standard udp configuration, create a separate config file with variables below and this config wouldn't affect working users):
```